dataminer/Functions/Video_Thumbnails/About_Video_Thumbnails.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: About_Video_Thumbnails
+---
+
+# About video thumbnails
+
+A video thumbnail is a built-in DataMiner feature that allows you to display a video feed or regularly refreshed image [in a Visio shape in Visual Overview](xref:Configuring_a_video_thumbnail_in_Visio) and [in a web component](xref:Configuring_a_video_thumbnail_in_a_web_component) in Dashboards and Low-Code Apps.
+
+The thumbnail is generated by a dedicated internal webpage hosted by the DataMiner Agent:
+
+```txt
+http(s)://<DMAIP>/VideoThumbnails/Video.htm?VideoServerParameters
```

Comment:
The <DMAIP> is not referring to the placeholder here so I wonder if we need to change it to the FQDN as we rarely work with IPs anymore.